seeds/blueberries_bnb.sql

```diff
@@ -45,10 +45,10 @@ CREATE TABLE bookings (
     on delete cascade
 );
 
-INSERT INTO users (email) VALUES ('blob@hotmail.com');
-INSERT INTO users (email) VALUES ('email2@hotmail.com');
-INSERT INTO users (email) VALUES ('email3@email.com');
-INSERT INTO users (email) VALUES ('email4@email.com');
+INSERT INTO users (email, password) VALUES ('blob@hotmail.com', 'scrypt:32768:8:1$D5hPxJWs7M5vRKiG$c7b58270a7aa94d2c46562cf7997242b2e125b64efc2e606334d6cec4bfa109476cf42834d6c6a2bc3a7e7e731c59652a14ea954d1f565796f6cd47aebd180bf');
```

Comment:
What's going on here?

Reply:
Clarified that these are the hash now in the seeds